internal/utils/bytes.go

```diff
@@ -89,7 +89,7 @@ func countOneBitsInByte(by byte) int {
 }
 
 // CountOneBits counts the number of 1 bit in byte array
-func CountOneBits(arr []byte) int {
+func CountOneBits(arr []byte) int64 {
```

Comment:
reason is to be system independent?

Reply:
No, so that easier to deal with the result of `Quroum` count result from https://github.com/harmony-one/harmony/pull/1717/files#diff-3a5c8ea459d421751fb70343be214598R32